knative-operator/deploy/resources/knativekafka/kafkachannel-v0.17.1.yaml

```diff
@@ -417,23 +417,6 @@ data:
   bootstrapServers: REPLACE_WITH_CLUSTER_URL
 ---
 apiVersion: v1
-kind: ConfigMap
```

Comment:
that sould have been removed on a different PR? :eyes:

Reply:
Yeah, https://github.com/openshift-knative/serverless-operator/pull/536

I am not doing things sequentially. I just rebased this PR on that.